crates/rover-client/src/query/partial/fetch.rs

```diff
@@ -0,0 +1,196 @@
+use crate::blocking::StudioClient;
+use crate::RoverClientError;
+use graphql_client::*;
+
+#[derive(GraphQLQuery)]
+// The paths are relative to the directory where your `Cargo.toml` is located.
+// Both json and the GraphQL schema language are supported as sources for the schema
+#[graphql(
+    query_path = "src/query/partial/fetch.graphql",
+    schema_path = ".schema/schema.graphql",
+    response_derives = "PartialEq, Debug, Serialize, Deserialize",
+    deprecated = "warn"
+)]
+/// This struct is used to generate the module containing `Variables` and
+/// `ResponseData` structs.
+/// Snake case of this name is the mod name. i.e. fetch_subgraph_query
+pub struct FetchSubgraphQuery;
+
+/// The main function to be used from this module. This function fetches a
```

Comment:
I would drop the first sentence of this doc comment. When you view this on docs.rs, it's a bit hard to figure out what `this module` is so it reads like a developer comment rather than a public facing API comment.

Reply:
Great point!